docs/5.0/preview/database-access.md

```diff
@@ -191,30 +286,140 @@ proxy_service:
   enabled: "no"
 ```
 
-Start the service:
+!!! tip
+    Single Teleport process can run multiple different services, for example
+    multiple database access proxies as well as run other services such an
+    SSH service or an application access proxy.
+
+Start the database service:
 
 ```sh
-$ teleport start --debug --config=/path/to/teleport-db.yaml
+$ teleport start -d --config=/path/to/teleport-db.yaml
 ```
 
-### Login
+### AWS Credentials
+
+When setting up Teleport database service with AWS RDS or Aurora, it must have
+an IAM role allowing it to connect to that particular database instance. An
+example of such policy is shown in the [AWS RDS/Aurora](#aws-rdsaurora-postgresql)
+section above. See [Creating and using an IAM policy for IAM database access](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html)
+in AWS documentation.
+
+Teleport database service uses default credential provider chain to find AWS
+credentials. See [Specifying Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials)
+for more information.
+
+## Connect
 
-After the setup, log into the cluster and see the registered database proxy:
+Once the database service has joined the cluster, login to see the available
+databases:
 
 ```sh
 $ tsh login --proxy=teleport.example.com:3080
 $ tsh db ls
+Name   Description Labels
+------ ----------- --------
+aurora AWS Aurora  env=dev
 ```
 
-### Connect
+Note that you will only be able to see databases your role has access to. See
+[RBAC](#rbac) section for more details.
 
-Log into the database and connect using `psql`:
+To connect to a particular database server, first retrieve credentials from
+Teleport using `tsh db login` command:
 
 ```sh
-$ tsh db login postgres
-$ psql "service=postgres-aurora user=<db-user> database=<db-name>"
+$ tsh db login aurora
 ```
 
+!!! tip
+    You can be logged into multiple databases simultaneously.
+
+You can optionally also specify the database name and the user to use by
+default when connecting to the database instance:
+
+```sh
+$ tsh db login --db-user=postgres --db-name=postgres aurora
```

Comment:
Updated to match example. 
```suggestion
$ tsh db login --db-user=postgres --db-name=aurora
```

Reply:
The current version is correct as it is actually. We're connecting to the database "postgres" within the database server "aurora".